Source/FieldSolver/WarpXPushFieldsEM_K.H

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2019-2020
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WarpXPushFieldsEM_K_h
+#define WarpXPushFieldsEM_K_h
+
+#include "Utils/WarpXConst.H"
+#include <AMReX.H>
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void damp_field(
+    amrex::Box tilebox,
```

Comment:
Instead of passing in the whole tilebox, can this instead be only the small end and big end values?